templates/zerver/help/send-a-message-in-a-different-language.md

```diff
@@ -0,0 +1,78 @@
+# Send a message in a different language
+
+To send a message in a different language you have to change your keyboard settings
+through your Operating System.
```

Comment:
Hmm, I am not sure we should capitalize operating system. It's not used as a proper noun here.